Migration/V2_0_0.md

```diff
@@ -0,0 +1,111 @@
+# v2.0.0 - Changelog & Migration Guide
+
+## Calls
+
+### AnyCall
+
+* `AnyCall.ResponseType` was renamed to `AnyCall.Parser`. This does also affect all subtypes.
+
+* Migrate:
+- use `AnyCall.Parser` instead of `AnyCall.ResponseType`
+
+### Call
+
+* `Call.ResponseType` was renamed to `Call.Parser`. This does also affect all subtypes.
+
+* Migrate:
+- use `YourCall.Parser` instead of `YourCall.ResponseType`
+
+### Convenience
+
+#### EmptyResponseParser
```

Comment:
Naming-vorschlag: Vielleicht NoContentParser? 
oder hast an besseren Vorschlag?

Reply:
Yes, `NoContentParser` is likely the better choice, as one primary use case is for 204 responses, which are called No Content. Will update!

Reply:
nice

Reply:
Done